startup.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 flask manage_db drop_all
```

Comment:
it is a bit couter-intutive that this script drops all databases no matter what.

Reply:
how is it counterintuitive?

Reply:
you run startup.sh and all your data will be deleted. A common practice is to be extra careful before permanently deleting data. See for example https://uxmovement.com/buttons/how-to-make-sure-users-dont-accidentally-delete/